scss/_footer.scss

```diff
@@ -54,10 +53,27 @@
     .item-link:hover {
       transform: scale(1.2);
     }
+    
+    .contact-icon {
+      width: 50px ;
+      height: 50px;
+      background-repeat: no-repeat;
+      background-size: contain;
+      margin: 10px 15px 0px 15px;
```

Comment:
Việc xét margin trong img thế này có làm việc click vào icon to ra hơn so với thực tế k?

Reply:
không nhé, vì t dùng img chứ không dùng fas